apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -367,22 +365,23 @@ protected static String getToken(CommandLine cmdLine) {
 
   protected static TablesClientFactory getTablesClientFactory(CommandLine cmdLine) {
     String token = getToken(cmdLine);
+    int tableMinAgeThresholdHours =
```

Comment:
Do you need a check whether the command line parameters are set prior to accessing them?